# IN MM ** IN MM
---
int16 x_pos
int16 y_pos
int16 z_pos
# IN MM ** IN MM